### PR TITLE
Updating cycle select placements

### PR DESCRIPTION
**Pull request by noahmanger**
- Adds a text string of the cycle to each section heading on candidate, committee and election page sections
- Moves the cycle select to the header of the election pages

Resolves https://github.com/18F/openFEC-web-app/issues/847

### CODE REVIEW

**Comment by noahmanger** [comment]
Ok this is good to go. https://github.com/18F/fec-style/pull/153 resolves @jenniferthibault 's one piece of design feedback.

File: templates/partials/committee/receipts-tab.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <div class="container">
     <div class="section__heading">
       <h2 class="heading__title" id="section-2-heading">
-        Money received by the <span class="term" data-term="Committee">committee</span>
+        Money received by the committee - {{ cycle|fmt_year_range }}
```
Comment (jmcarp) [inline templates/partials/committee/receipts-tab.html]: Just curious--why drop the glossary lookup here? I'm not clear on when we want to wrap terms in glossary lookups and when we don't.

Comment (noahmanger) [inline templates/partials/committee/receipts-tab.html]: Good question. I think it makes sense to not make words in headings glossary links. @emileighoutlaw ?

**Comment by noahmanger** [comment]
Conflicts resolved.

**Pull request merged**